.github/workflows/storybook.yml

```diff
@@ -16,8 +16,7 @@ env: # Set environment variables for every job and step in this workflow
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write # for dependabot
+    permissions: write-all # for dependabot
```

Comment:
Would just adding `pages: write` be enough then?

Reply:
🤔 valid question. So you'd recommend `pull-requests` and `pages`, right? We can try it

Reply:
Yeah, I'd prefer the least privilege approach :)